scripts/UpdateRepos.ps1

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env powershell
+
+<#
+.SYNOPSIS
+    Updates each repo Universe builds to new dependencies.props
+.PARAMETER Source
+    The NuGet package source to find the lineup on.
+.PARAMETER LineupID
+    The ID of the Lineup to determine which versions to use.
+.PARAMETER LineupVersion
+    The version of the Lineup to be used.
+#>
+[cmdletbinding(SupportsShouldProcess = $true)]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$Source,
+    [Parameter(Mandatory=$true)]
+    [string]$LineupID,
+    [Parameter(Mandatory=$true)]
+    [string]$LineupVersion
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2
+
+Import-Module "$PSScriptRoot/common.psm1" -Scope Local -Force
+
+$RepoRoot = Resolve-Path "$PSScriptRoot\.."
+$ModuleDirectory = Join-Path $RepoRoot "modules"
+
+try {
+    Push-Location $ModuleDirectory
+
+    # Init all submodules
+    Invoke-Block { & git submodule update --init }
+
+    $submodules = Get-Submodules $ModuleDirectory
+
+    $submodules `
+    | % {
+        try {
+            Push-Location $_.path
+
+            $depsFile = Join-Path (Join-Path $($_.path) "build") "dependencies.props"
+            $gitModules = Join-Path $RepoRoot ".gitmodules"
+            # Move to latest commit on tracked branch
+            $branch = $(git config -f $gitModules --get submodule.modules/$($_.module).branch )
+            Write-Output "Branch: $branch"
+
+            Invoke-Block { & git checkout --quiet $branch }
+
+            Invoke-Block { & .\run.ps1 upgrade deps --source $Source --id $LineupID --version $LineupVersion --deps-file $depsFile }
+            Invoke-Block { & git add $depsFile }
+            Invoke-Block { & git commit --quiet --author "ASP.NET Push Bot <asplab@microsoft.com>" -m "[auto]Update dependencies.props" }
```

Comment:
Can you change the message to be consistent with the auto-updated tag we've added in the other universe update scripts?
```
-m "Update dependencies.props`n`n[auto-updated: dependencies]"
```